aten/src/ATen/nn_parse.py

```diff
@@ -290,6 +282,12 @@ def backward_declaration(base, thnn_functions):
 
     arguments += unique_args([output_arguments(f) for f in thnn_functions])
 
+    if 'upsample' in base['name']:
```

Comment:
If we're adding a special case for this function, can we add a comment to the source nn.yaml file pointing out that there is a special case?